shims/spark311/src/main/scala/com/nvidia/spark/rapids/shims/spark311/SparkShimServiceProvider.scala

```diff
@@ -22,7 +22,8 @@ object SparkShimServiceProvider {
   // keep 3.1.0 snapshot version for now until 3.1.1 is released
   val VERSION310 = SparkShimVersion(3, 1, 0)
   val VERSION = SparkShimVersion(3, 1, 1)
-  val VERSIONNAMES = Seq(s"$VERSION310-SNAPSHOT", s"$VERSION", s"$VERSION-SNAPSHOT")
+  val VERSIONNAMES: Seq[String] = Seq(VERSION310, VERSION)
+    .flatMap(v => Seq(s"$v", s"$v-SNAPSHOT"))
```

Comment:
Please revert this change.  It's going to conflict with the change coming in #1832 and isn't necessary for this PR.